slides.md

```diff
@@ -385,45 +385,99 @@ export async function comparePassword(password, hash) {
 
 ---
 
-# A03: Injection
+# A03: Injection 💉
 
 <div class="dense">
 
 - Injections are a form of attack where a malicious payload is able to effectively inject an arbitrary bit of query or code on the target server
-- Common targets: SQL, NoSQL, ORM, LDAP, JS eval
+- Injections can result in data loss or corruption, lack of accountability, or denial of access. Injections can sometimes lead to complete host takeover.
+- Common targets: **SQL, NoSQL, ORM, LDAP, JS eval**
 
 </div>
 
 ---
 
-# A03 Injection (2): Common vulnerabilities
+# A03 Injection 💉: Common vulnerabilities
 
 <div class="dense">
 
-- Lack of validation or sanitization of user input
-- Dynamic queries or non-parameterized calls without context-aware escaping are used directly in the interpreter
-- Hostile data is used within object-relational mapping (ORM) search parameters to extract additional, sensitive records
+- Lack of validation or **sanitization** of user input
+- Dynamic queries or non-parameterized calls without context-aware **escaping** are used directly in the interpreter
+- Hostile data is used within object-relational mapping (ORM) **search parameters** to extract additional, sensitive records
 
 </div>
 
 ---
 
-# A03 Injection (3): How to Prevent
+# A03 Injection 💉: Attack
 
 <div class="dense">
 
-- Prefer using a safe API that sanitizes input
-- Escape special characters using the specific escape syntax for that interpreter
+- Run the server for step 3 (`cd src/a03-injection`, `npm start`)
+- In Postman, run the query for `A03: Injection`. Observe the data for `name: "alice"` being returned
+- Try to inject a value in the query param `name` that retrieves all the customers
+
+</div>
+
+---
+
+# A03 Injection 💉: Solution
+
+<div class="dense">
+
+- The query param value `' OR '1'='1` takes advantage of the unsafe string concatenation to create this query
```

Comment:
I would include an example of the actual attacking query in postman so people can run it maybe